app/src/main/res/layout/row_item_description.xml

```diff
@@ -1,52 +1,89 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  android:layout_width="match_parent"
+  android:layout_height="wrap_content"
+  android:layout_marginTop="12dp"
+  android:layout_marginBottom="@dimen/activity_margin_horizontal"
+  android:layout_marginStart="@dimen/activity_margin_horizontal"
+  android:layout_marginEnd="@dimen/activity_margin_horizontal"
+  android:layout_marginLeft="@dimen/activity_margin_horizontal"
```

Comment:
don't use left/right with start/end

Reply:
My bad, effect of copying it from other layouts :P

Reply:
it happens, this is why we have reviews after all